server/channels/app/platform/service.go

```diff
@@ -275,6 +275,7 @@ func New(sc ServiceConfig, options ...Option) (*PlatformService, error) {
 
 	// Needed before loading license
 	ps.statusCache, err = ps.cacheProvider.NewCache(&cache.CacheOptions{
+		Name:           "Status",
```

Comment:
Would it make sense to add a validity check on `Name != ""` to avoid sneaky regressions?

Reply:
It's already there in my redis PR. I don't want to duplicate it here :)